x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.test.tsx

```diff
@@ -998,37 +1003,66 @@ describe('when on the endpoint list page', () => {
 
   describe('when the more actions column is opened', () => {
     const generator = new EndpointDocGenerator('seed');
-    let hostInfo: HostInfo;
+    let hostInfo: HostInfo[];
     let agentId: string;
     let agentPolicyId: string;
     let renderResult: ReturnType<AppContextTestRender['render']>;
 
+    // 2nd endpoint only has isolation capabilities
     const mockEndpointListApi = () => {
-      const { data: hosts } = mockEndpointResultList();
-      hostInfo = {
-        host_status: hosts[0].host_status,
-        metadata: {
-          ...hosts[0].metadata,
-          Endpoint: {
-            ...hosts[0].metadata.Endpoint,
-            state: {
-              ...hosts[0].metadata.Endpoint.state,
-              isolation: false,
+      const { data: hosts } = mockEndpointResultList({ total: 2 });
+      hostInfo = [
+        {
+          host_status: hosts[0].host_status,
+          metadata: {
+            ...hosts[0].metadata,
+            Endpoint: {
+              ...hosts[0].metadata.Endpoint,
+              capabilities: RESPONDER_CAPABILITIES,
```

Comment:
instead of using the actual array, can you spread it into a new one?

```
[ ...RESPONDER_CAPABILITIES ]
```

Reply:
I can. Is there a benefit for doing this?